src/settings.js

```diff
@@ -85,6 +85,23 @@ export default {
       externalLink: 'https://dev.to',
       bodyComponentName: 'DevtoBody',
 
+      nightMode: {
+        titleBackgroundColor: DEFAULT_NIGHTMODE_BACKGROUND,
+        titleFontColor: 'ffffff',
+        loadingColor: 'ffffff',
+      },
+    },
+    lobsters: {
+      dataUrl: '/proxy/hottest.json',
+      title: 'Lobsters',
+      icon: ['fa', 'anchor'],
+      titleFontColor: 'ffffff',
+      titleBackgroundColor: '9E271B',
+      loadingColor: '9E271B',
```

Comment:
let's go lowercase here like all the other colors. I'll probably review the colors once I get the chance to check it out locally.